pkgs/by-name/so/softether/package.nix

```diff
@@ -1,22 +1,23 @@
 {
   lib,
   stdenv,
-  fetchurl,
+  fetchFromGitHub,
   openssl,
   readline,
   ncurses,
   zlib,
   dataDir ? "/var/lib/softether",
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "softether";
-  version = "4.38";
-  build = "9760";
+  version = "4.41";
 
-  src = fetchurl {
-    url = "https://github.com/SoftEtherVPN/SoftEtherVPN_Stable/releases/download/v${version}-${build}-rtm/softether-src-v${version}-${build}-rtm.tar.gz";
-    sha256 = "0d8zahi9lkv72jh8yj66pwrsi4451vk113d3khzrzgbic6s2i0g6";
+  src = fetchFromGitHub {
+    owner = "SoftEtherVPN";
+    repo = "SoftEtherVPN_Stable";
+    rev = "v4.41-9782-beta";
```

Comment:
Shouldn't the `version` attribute equal to that ?

Reply:
Yes, it should.

Reply:
I dont mind to do so but the package version before also did not bother with that ... and upstream is kinda weard also